assets/src/room.ts

```diff
@@ -66,17 +71,21 @@ export class Room {
         },
         onConnectionError: setErrorMessage,
         onJoinSuccess: (_peerId, peersInRoom) => {
-          this.localAudioStream
-            ?.getTracks()
-            .forEach((track) =>
-              this.webrtc.addTrack(track, this.localAudioStream!, {})
-            );
+          let audio_ids = this.localAudioStream?.getTracks().map((track) =>
+            this.webrtc.addTrack(track, this.localAudioStream!, {
+              active: true,
+            })
+          );
+
+          if (audio_ids != undefined) {
+            this.localAudioTrackIds = this.localAudioTrackIds.concat(audio_ids);
+          }
```

Comment:
This is quite incocsistent with tthe way `localVideoTrackId`
```suggestion
          let audio_ids = this.localAudioStream?.getTracks().forEach((track) =>
            const trackId = this.webrtc.addTrack(track, this.localAudioStream!, {
              active: true,
            });
            this.localAudioTrackIds.push(trackId);
          );
```